src/kiwixserve.cpp

```diff
@@ -39,7 +39,7 @@ void KiwixServe::run()
     std::string kiwixServeCmd = appendToDirectory(
         removeLastPathElement(getExecutablePath(true)),
         KIWIXSERVE_CMD);
-    if (fileExists(kiwixServeCmd)) {
+    if (fileExistsAndReadable(kiwixServeCmd)) {
```

Comment:
Here we don't need that `kiwixServeCmd` is readable.
Just that the file exists. (If we want further check, we should check it is executable)

Reply:
same with `ARIA2_CMD`?

Reply:
yes